examples/import-boss/generators/import_restrict.go

```diff
@@ -192,8 +192,16 @@ func (importRuleFile) VerifyFile(f *generator.File, path string) error {
 	}
 
 	if rules == nil {
-		// No restrictions on this directory.
-		return nil
+		// No restrictions on this directory, double check under vendor.  Make an assumption about running in kube
+		vendorPath := strings.Replace(path, "/go/src/k8s.io/", "/go/src/k8s.io/kubernetes/vendor/k8s.io/", -1)
```

Comment:
I can't seem to find a place that indicates the directory that the supporting files were found in, do the path always ends up k8s.io/kubernetes without vendor in it.

As I noted in the description, I know this isn't right, but this is the spot I need to know where to look for the .Import-path and I'm looking for a recommendation of what to use to find it.